packages/flutter/test/widgets/draggable_scrollable_sheet_test.dart

```diff
@@ -1404,22 +1404,21 @@ void main() {
     await tester.pumpWidget(boilerplate);
     expect(controller.isAttached, true);
     expect(controller.size, isNotNull);
-    });
+  });
```

Comment:
(I was looking at these tests initially - no actually changes here, just fixing the whitespace)